README.md

```diff
@@ -24,3 +24,13 @@ $ python src/train.py --model neuralbow trained_models \
     resources/data/python/final/jsonl/valid \
     resources/data/python/final/jsonl/test
 ```
+
+### AST parsing
+
+For AST parsing we use [tree-sitter](https://github.com/tree-sitter/tree-sitter).
+To work with parsers, do the following:
+
+1. Clone the appropriate tree-sitter parser in `src/parsing/tree-sitter-languages`
```

Comment:
in `src/processing/tree-sitter-languages`?

Reply:
Should be "to `src/parsing/...`"